lib/Zonemaster/Backend/Config.pm

```diff
@@ -138,6 +138,8 @@ sub parse {
     $obj->_set_ZONEMASTER_age_reuse_previous_test( '600' );
     $obj->_add_LANGUAGE_locale( 'en_US' );
     $obj->_add_public_profile( 'default', undef );
+    #$obj->_set_METRICS_statsd_host( undef );
```

Comment:
What is the purpose of this commented line? Either remove it, remove `#` or add a comment that explains why it is there but commented.

Reply:
None, it has been removed